server.js

```diff
@@ -0,0 +1,309 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const querystring = require('querystring');
+
+const PORT = process.env.PORT || 8080;
+
+const Users = {
+  gene: 'bean'
+};
+
+const server = http.createServer((req, res) => {
+  if (req.method === 'GET') {
+    let uri;
```

Comment:
If you start out with:
`let uri = req.url;`
Then you won't need the else statement directly below.